server/graphql_schemas/utils/helpers.py

```diff
@@ -101,6 +101,26 @@ def build_event(event, invitees):
     return event
 
 
+def validate_event_dates(input):
+    """
+        Validate date fields
+         :param input:
+    """
+
+    today_date = int(parser.parse(str(datetime.datetime.now())).strftime("%s"))
```

Comment:
We may not need all of the type conversion that we have here. `input.get('start_date')` already returns a datetime object. Same as `input.get('end_date')`. One consideration we would need to have here is that `today_date` must be timezone-aware just like `start_date` and `end_date to ensure effective date comparison. Our server may be hosted in a completely different timezone and `datetime.now()` will return local datetime in that timezone. To ensure that we have an aware `today_date`, we can have:
```
event_tz = input.get('start_date').tzinfo
today_date = datetime.datetime.now(event_tz)
```

See here for more ideas on date comparison: https://www.geeksforgeeks.org/comparing-dates-python/